Sources/Helpers/OAAppSettings.m

```diff
@@ -4037,8 +4037,21 @@ - (instancetype) init
         [_profilePreferences setObject:_locationIcon forKey:@"location_icon"];
 
         _appModeOrder = [OACommonInteger withKey:appModeOrderKey defValue:0];
+        [_appModeOrder setModeDefaultValue:@0 mode:OAApplicationMode.DEFAULT];
```

Comment:
NSArray *modes = @[
    OAApplicationMode.DEFAULT,
    OAApplicationMode.CAR,
...
];
and for (NSInteger i = 0; i < modes.count; i++) {}